apps/frontend/src/app/shared/feedback/error/error.component.ts

```diff
@@ -15,6 +15,7 @@ export class ErrorComponent {
   @Input() action = 'Try again'
   @Input() route = '/'
   @Input() callback: (() => void) | null = null
+  @Input() showAction = true
```

Comment:
What does this help? Button is rendered neither way? 🤔

Reply:
there is a default text but i just realized that this part of the code is bullshit anyways